packages/eslint-config-airbnb-base/README.md

```diff
@@ -15,8 +15,8 @@ Our default export contains all of our ESLint rules, including ECMAScript 6+. It
 1. Ensure packages are installed with correct version numbers by running:
   ```sh
   (
-    export PKG=eslint-config-airbnb-base;
-    npm info "$PKG" peerDependencies --json | command sed 's/[\{\},]//g ; s/: /@/g' | xargs npm install --save-dev "$PKG"
+    PKG=eslint-config-airbnb-base
+    npm info "$PKG" peerDependencies --json | sed 's/[\{\},]//g ; s/: /@/g' | xargs npm install --save-dev "$PKG"
```

Comment:
Just to make sure we're on the same page, from the Dash manual:

> Execute the specified command but ignore shell functions when searching for it.  (This
> is useful when you have a shell function with the same name as a builtin command.)

Then I should change this to `command npm`, `command xargs`, ... in case a user defines their own functions for those as well? If a user has their own sed binary, they probably want to use it. In which case, they've either intentionally overridden the builtin and command will be unwanted or they haven't and PATH will be used so command is unneeded. Regardless, a user can't expect to override shell standards without consequence. They should choose a unique name instead. It seems foolish and obfuscating to attempt to code for someone who has sabotaged their own shell so severely.